src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -358,6 +403,37 @@ public bool Disconnect(DisconnectParams disconnectParams)
                 return false;
             }
 
+            // If we are disconnecting all existing connections for the provided URI
+            if (disconnectParams.Type == ConnectionType.AllConnections)
+            {
+                foreach (KeyValuePair<ConnectionType, DbConnection> entry in info.ConnectionTypeToConnectionMap)
+                {
+                    entry.Value.Close();
```

Comment:
Should this be wrapped in a try/catch to handle the case where closing the connection fails for some reason?